coral/src/app/features/components/table-filters/useTableFiltersValues.tsx

```diff
@@ -0,0 +1,48 @@
+import { useSearchParams } from "react-router-dom";
+import { AclType } from "src/domain/acl/acl-types";
+import { RequestStatus } from "src/domain/requests/requests-types";
+
+type UseTableFiltersValuesParams =
+  | {
+      defaultTopic?: string;
+      defaultEnvironment?: string;
+      defaultAclType?: AclType | "ALL";
+      defaultStatus?: RequestStatus;
+      defaultTeam?: string;
+    }
+  | undefined;
+
+const useTableFiltersValues = (
+  defaultValues: UseTableFiltersValuesParams = {}
+) => {
+  const [searchParams] = useSearchParams();
+  const {
+    defaultTopic = "",
+    defaultEnvironment = "ALL",
+    defaultAclType = "ALL",
+    defaultStatus = "ALL",
+    defaultTeam = "ALL",
+  } = defaultValues;
+
+  const currentTopic = searchParams.get("topic") ?? defaultTopic;
+  const currentEnvironment =
+    searchParams.get("environment") ?? defaultEnvironment;
+  const currentAclType =
+    (searchParams.get("aclType") as AclType) ?? defaultAclType;
+  const currentStatus =
+    (searchParams.get("status") as RequestStatus) ?? defaultStatus;
+  const currentTeam = searchParams.get("team") ?? defaultTeam;
+  const showOnlyMyRequests =
+    searchParams.get("showOnlyMyRequests") === "true" ? true : undefined;
+
+  return {
```

Comment:
Looking at how it's used now, I think we can get rid of the "current" prefix! Since there's only one e.g. "environment" (the one from the search params) there's no need to mark it as "current". It even implies a bit that there could be a different "environment" (aka: not the current one) somewhere - which there isn't.